configure.py

```diff
@@ -813,11 +813,11 @@
     default=None,
     help='do not install the bundled npm (package manager)')
 
-parser.add_argument('--without-corepack',
+parser.add_argument('--with-corepack',
     action='store_true',
-    dest='without_corepack',
+    dest='with_corepack',
     default=None,
-    help='do not install the bundled Corepack')
+    help='do install the bundled Corepack')
```

Comment:
```suggestion
    help='bundle Corepack')
```

Reply:
```suggestion
    help='do install the bundled Corepack (experimental, will be removed without notice)')
```

Reply:
"do install" at the front of the sentence doesn't read like proper english to me, and there is no "bundled corepack" anymore _to_ install - this flag causes it to be bundled.

Reply:
I agree that "bundled Corepack" is weird, maybe "vendored" would be more appropriate?

Reply:
the problem is the "the" - there *is* no "bundled corepack" anymore. the flag causes corepack to BE bundled, and "bundle" is a verb, which is why "bundle corepack" seems like the best/proper phrasing to me (with or without the experimental note)